src/conversions/num_bigint.rs

```diff
@@ -128,6 +166,32 @@ macro_rules! bigint_conversion {
                     }
                 }
             }
+
+            #[cfg(Py_LIMITED_API)]
```

Comment:
where will be the proper places to put the limited api implementation? Like now or a new file or module?

Reply:
It looks like the only thing which changes here is the implementation of how to get `n_bytes`, so I think we can do something like:

```rust
let n_bytes = cfg_if::cfg_if! {
    if #[cfg(not(Py_LIMITED_API))] {
        // fast path
    } else { 
        // slow path
    }
}
```